src/assets/styles/themes.css

```
@@ -1,5 +1,6 @@
 :root {
   --background-color: 255, 255, 255;
+  --secondary-background-color: 233, 235, 238;
```

```suggestion
  --background-secondary-color: 233, 235, 238;
```

Modifiers/Variants after. Rename the variable for the dark classes and tailwind config as well.

---

Variable renamed